datasets/textvqa/README.md

```diff
@@ -0,0 +1,203 @@
+---
+annotations_creators:
+- crowdsourced
+language_creators:
+- crowdsourced
+languages:
+- en
+licenses:
+- cc-by-4.0
+multilinguality:
+- monolingual
+pretty_name: TextVQA
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- question-answering
```

Comment:
`question-answering` means extractive-qa for text in `transformers` and on the Hub. In particular each task category refers to a pipeline type in `transformers`.

Therefore I would create a new task category for `visual-question-answering` instead of having it under `question-answering`

Feel free to open a PR to add it to https://github.com/huggingface/hub-docs/blob/main/js/src/lib/interfaces/Types.ts :) cc @osanseviero would it make sense in your opinion ?

Reply:
I just noticed that `visual-question-answering` is already there: https://github.com/huggingface/hub-docs/blob/f9e8ec8f882ddb2b0517e97afc998f9cf398953b/js/src/lib/interfaces/Types.ts#L549